### PR TITLE
Add desktop & mobile design for team members

### PR DESCRIPTION
**Pull request by tlorent**

### CODE REVIEW

File: components/info-bar/InfoBar.jsx [hunk 1]
```diff
@@ -1,20 +1,26 @@
 import React from 'react';
+import propTypes from 'prop-types';
```
Comment (boris-arkenaar) [inline components/info-bar/InfoBar.jsx]: It is actually `PropTypes`.

File: components/team-member/team-member.scss [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      margin-right: 54px;
+    }
+  }
+
+  &__name,
+  &__number,
+  &__employer,
+  &__start-date {
+    color: $color-monochrome-6-alt-1;
+    font-family: ProximaNova;
```
Comment (boris-arkenaar) [inline components/team-member/team-member.scss]: This is already the default font, is it not?

File: components/team-members-overview/TeamMembersOverview.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import './team-members-overview.scss';
+
+class TeamMembersOverview extends React.Component {
+  state = {
+    test: 'test'
+  };
+
+  render() {
+    return (
+      <React.Fragment>
```
Comment (boris-arkenaar) [inline components/team-members-overview/TeamMembersOverview.jsx]: You have one root `div`. No `Fragment` needed.

File: components/team-members-overview/TeamMembersOverview.jsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+                placeholder="Sort by:"
+              />
+              <img
+                src="../../../../static/icons/arrow-down.svg"
+                alt="arrow pointing down"
+                className="team-members-overview__arrow-down"
+              />
+            </div>
+          </div>
+          <section className="team-members-section">
```
Comment (boris-arkenaar) [inline components/team-members-overview/TeamMembersOverview.jsx]: Keep it an 'element' of your 'block': `team-members-overview__section`, or `team-members-overview__team-member`?

File: components/team-member/TeamMember.jsx [hunk 5]
```diff
@@ -14,42 +18,59 @@ const TeamMember = () => (
...
         </p>
-        <p className="team-member__subtitle">
+        <p className="team-member__number-subtitle">
           Employee number
         </p>
       </div>
-      <div className="team-member__box">
+      <div className={`
+        team-member__box
+        team-member__box--employer${isDetailVisible ? '--visible' : ''}
```
Comment (boris-arkenaar) [inline components/team-member/TeamMember.jsx]: Use only one modifier per per css class name.
```
team-member__box
team-member__box--employer
team-member__box--${isDetailVisible ? 'visible' : 'hidden'}
```
Besides: is `employer` a modifier?

File: components/team-member/TeamMember.jsx [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+      tabIndex="0"
+      onKeyPress={onExpand}
+      onClick={onExpand}
+      className={`
+        team-member__arrow-down
+        team-member__arrow-down${isDetailVisible ? '--rotated' : ''}
+      `}
+    >
+      <img
+        src="../../../../static/icons/arrow-down-2.svg"
```
Comment (larstroost) [inline components/team-member/TeamMember.jsx]: Works as well if you type: src="static/icons/arrow-down-2.svg

File: components/team-member/team-member.scss [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+      }
+
+      @include tablet {
+        display: block;
+        margin: 0;
+      }
+    }
+  }
+
+  &__name,
```
Comment (larstroost) [inline components/team-member/team-member.scss]: Maybe you could give them a generic name to combine all four.

**Pull request merged**